firmware/tunerstudio/rusefi.input

```diff
@@ -1854,9 +1856,14 @@ cmd_stop_engine              = "w\x00\x99\x00\x00"
 		field = "Injection Mode",						crankingInjectionMode
 		field = "Base Fuel Pulse Width", 				cranking_baseFuel
 		
+	dialog = crankingIAC, "IAC"
+		field = "Cranking IAC position",					crankingIACposition
+		field = "After cranking IAC taper duration",	afterCrankingIACtaperDuration
+		field = "Override IAC Multiplier for Cranking",	overrideCrankingIacSetting	
+		field = "Force parking every restart",			stepperForceParkingEveryRestart, useStepperIdle		
 		
 	dialog = crankingIgnition, "Ignition"
-		field = "Advance", 								crankingTimingAngle
+		field = "Advance", 								crankingTimingAngle, {useSeparateAdvanceForCranking == 0}
```

Comment:
We want to disable this 'advange angle' setting if a separate advance table is selected for cranking.